projects/laji/src/app/+theme/theme.component.html

```diff
@@ -8,127 +8,127 @@ <h1 class="mt-6">
   <div class="d-flex flex-wrap">
-    <a [routerLink]="[({fi: '/lajiluettelo', sv: '/artlistan', en: '/checklist'} | multiLang)]" tabindex="-1" class="navbox">
+    <a [routerLink]="[({fi: '/lajiluettelo', sv: '/artlistan', en: '/checklist'} | multiLang)]" tabindex="0" class="navbox">
       <h5>{{ 'navigation.checklist' | translate }}</h5>
       <p>{{ 'theme.checklist.intro' | translate }}</p>
     </a>
   </div>
 
   <h4><span class="glyphicon glyphicon-map-marker"></span> {{ 'theme.place.title' | translate }}</h4>
   <div class="d-flex flex-wrap">
-    <a [routerLink]="['/map'] | localize" tabindex="-1" class="navbox">
+    <a [routerLink]="['/map'] | localize" tabindex="0" class="navbox">
       <h5>{{ 'navigation.map' | translate }}</h5>
       <p>{{ 'theme.map.intro' | translate }}</p>
     </a>
-    <a [routerLink]="['/ykj'] | localize" tabindex="-1" class="navbox">
+    <a [routerLink]="['/ykj'] | localize" tabindex="0" class="navbox">
       <h5>{{ 'navigation.ykj' | translate }}</h5>
       <p>{{ 'theme.ykj.intro' | translate }}</p>
     </a>
-    <a [routerLink]="['/emk'] | localize" tabindex="-1" class="navbox">
+    <a [routerLink]="['/emk'] | localize" tabindex="0" class="navbox">
       <h5>{{ 'navigation.emk' | translate }}</h5>
       <p>{{ 'theme.emk.intro' | translate }}</p>
     </a>
   </div>
 
   <h4><span class="glyphicon glyphicon-search"></span> {{ 'theme.research.main' | translate }}</h4>
   <div class="d-flex flex-wrap">
-    <a [routerLink]="['/about/2272'] | localize" tabindex="-1" class="navbox">
+    <a [routerLink]="['/about/2272'] | localize" tabindex="0" class="navbox">
       <h5>{{ 'navigation.research' | translate }}</h5>
       <p>{{ 'theme.research.intro' | translate }}</p>
     </a>
-    <a [routerLink]="['/theme/luomusgrc'] | localize" tabindex="-1" class="navbox">
+    <a [routerLink]="['/theme/luomusgrc'] | localize" tabindex="0" class="navbox">
       <h5>{{ 'theme.grc.title' | translate }}</h5>
       <p>{{ 'theme.grc.intro' | translate }}</p>
     </a>
-    <a [routerLink]="['/theme/kerttu'] | localize" tabindex="-1" class="navbox">
+    <a [routerLink]="['/theme/kerttu'] | localize" tabindex="0" class="navbox">
       <h5>{{ 'theme.kerttu' | translate }}</h5>
       <p>{{ 'theme.kerttu.intro' | translate }}</p>
     </a>
-    <a [routerLink]="['/theme/protax'] | localize" *lajiForTypes="['dev']" tabindex="-1" class="navbox">
+    <a [routerLink]="['/theme/protax'] | localize" *lajiForTypes="['dev']" tabindex="0" class="navbox">
       <h5>{{ 'theme.protax' | translate }}</h5>
       <p>{{ 'theme.protax.intro' | translate }}</p>
     </a>
-    <a [routerLink]="['/theme/publications'] | localize" tabindex="-1" class="navbox">
+    <a [routerLink]="['/theme/publications'] | localize" tabindex="0" class="navbox">
       <h5>{{ 'finbif-bib.title' | translate }}</h5>
       <p>{{ 'finbif-bib.intro' | translate }}</p>
     </a>
```

Comment:
`a` is tab accessible even without tabindex. This doesn't matter much since having it here doesn't affect the elements in any way, but  it's redundant.

Reply:
fixd